internal/gatewayapi/filters.go

```diff
@@ -76,7 +76,7 @@ func (t *Translator) ProcessHTTPFilters(parentRef *RouteParentContext,
 		RuleIdx:      ruleIdx,
 		HTTPFilterIR: &HTTPFilterIR{},
 	}
-
+	filterIdx := 0
 	for i := range filters {
```

Comment:
why not just use i

Reply:
> why not just use i

Yes, I can use 'i' if I don't need to match the value with the index of Mirror list.